src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -877,7 +877,10 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 							if (value is SystemTypeValue systemTypeValue) {
 								foreach (var stringParam in methodParams[1].UniqueValues ()) {
 									if (stringParam is KnownStringValue stringValue) {
-										MarkMethodsOnTypeHierarchy (ref reflectionContext, systemTypeValue.TypeRepresented, m => m.Name == stringValue.Contents, bindingFlags);
+										if ((bindingFlags & BindingFlags.IgnoreCase) == BindingFlags.IgnoreCase || (int) bindingFlags > 255)
```

Comment:
Nit: I would wrap this in a function - to make it more readable. Something like `BindingFlagsAreSupported` ... currently the condition looks like magic. In the separate function we can add a comment to list all of the supported cases to make it clear why we "filter" out the rest.